server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -117,7 +119,7 @@ public RemoteFsTranslog(
     }
 
     public static void download(TranslogTransferManager translogTransferManager, Path location) throws IOException {
-
+        logger.info("Downloading translog files from remote for shard {} ", translogTransferManager.getShardId());
```

Comment:
Why don't we take `shardId` as an additional parameter to `download` if `TranslogTransferManager` does not have any used of it?

Reply:
Since we are creating `TranslogTransferManager` per shard, it makes sense to have it in there.  By this we would be able to add shard details in other places as well .

Reply:
Should we have the log level as trace or debug?

Reply:
I have added `shard id` in critical logs now in transfer manager.